experiments/benchmarks/richards.jl

```diff
@@ -52,17 +51,15 @@ import Profile, ProfileCanvas
 const FT = Float64;
 
 regridder_type = :InterpolationsRegridder
+extrapolation_bc =
+    (Interpolations.Periodic(), Interpolations.Flat(), Interpolations.Flat())
```

Comment:
I set these to be the defaults already, but I thought it was good practice to still supply them so that people understand how to change them if they want. But I guess the doc string for the functions are enough for that, and I can remove them here.